tools/flang1/flang1exe/accpp.c

```diff
@@ -701,6 +701,20 @@ print_and_check(FILE *ff, char *str, char end_c)
     fputc(end_c, ff);
 }
 
+extern bool g_list_macros;
+void acc_display_macros()
+{
+  for (unsigned int i = 1; i<next_hash; ++i) {
+    char* name = &deftab[hashrec[i].name];
+    char* value = &deftab[hashrec[i].value];
+    if((strcmp(name,"__LINE__")==0) ||
+       (strcmp(name,"__DATE__")==0) ||
+       (strcmp(name,"__FILE__")==0) ||
+       (strcmp(name,"__TIME__")==0)) continue;
+    printf("%s : %s\r\n",name,value);
```

Comment:
there is a good chance it would be the only use of `\r\n` in the entire file. This reminds me my DOS years, but that was more than 30 years ago. `\n` is enough to end a line.